### PR TITLE
Docs: Move configuration information to top level, and rename section to Configure.

### PR DESCRIPTION
**Pull request by osg-grafana**

### CODE REVIEW

**Comment by osg-grafana** [comment]
@jdbaldry, When I was doing this manually, I saw that link after link the build got better so I thought I was making progress. However, the doc validator seems to have me changing them again, fortunately in much less time. However, I am not sure why the local build keep improving when I fixed all these links manually.

File: CHANGELOG.md [hunk 1]
```diff
@@ -356,7 +356,7 @@ Querying with using `{__mimir_storage__="ephemeral"}` selector no longer works.
     * Renaming `blocks_storage_backend` key to `storage_backend`.
     * For Azure/S3:
       * Renaming `blocks_storage_(azure|s3)_*` configurations to `storage_(azure|s3)_*`.
-      * If `ruler_storage_(azure|s3)_*` and `alertmanager_storage_(azure|s3)_*` keys were different from the `block_storage_*` ones, they should be now provided using CLI flags, see [configuration reference](https://grafana.com/docs/mimir/v2.3.x/operators-guide/configure/reference-configuration-parameters/) for more details.
+      * If `ruler_storage_(azure|s3)_*` and `alertmanager_storage_(azure|s3)_*` keys were different from the `block_storage_*` ones, they should be now provided using CLI flags, see [configuration reference](https://grafana.com/docs/mimir/v2.3.x/configure/reference-configuration-parameters/) for more details.
```
Comment (osg-grafana) [inline CHANGELOG.md:359]: Same thing here. Will come back to this to revert it.

File: docs/sources/helm-charts/mimir-distributed/migration-guides/migrate-from-single-zone-with-helm.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ weight: 10
 
 # Migrate from single zone to zone-aware replication
 
-This document explains how to migrate stateful components from single zone to [zone-aware replication](/docs/mimir/v2.6.x/operators-guide/configure/configure-zone-aware-replication/) with Helm. The three components in question are the [alertmanager](/docs/mimir/v2.6.x/operators-guide/architecture/components/alertmanager/), the [store-gateway](/docs/mimir/v2.6.x/operators-guide/architecture/components/store-gateway/) and the [ingester](/docs/mimir/v2.6.x/operators-guide/architecture/components/ingester/).
+This document explains how to migrate stateful components from single zone to [zone-aware replication](/docs/mimir/v2.6.x/configure/configure-zone-aware-replication/) with Helm. The three components in question are the [alertmanager](/docs/mimir/v2.6.x/operators-guide/architecture/components/alertmanager/), the [store-gateway](/docs/mimir/v2.6.x/operators-guide/architecture/components/store-gateway/) and the [ingester](/docs/mimir/v2.6.x/operators-guide/architecture/components/ingester/).
```
Comment (osg-grafana) [inline docs/sources/helm-charts/mimir-distributed/migration-guides/migrate-from-single-zone-with-helm.md:10]: This file will be removed in a subsequent PR. Please ignore it here.

Comment (osg-grafana) [inline docs/sources/helm-charts/mimir-distributed/migration-guides/migrate-from-single-zone-with-helm.md:10]: I need to revisit this.

**Comment by osg-grafana** [comment]
> @jdbaldry, When I was doing this manually, I saw that link after link the build got better so I thought I was making progress. However, the doc validator seems to have me changing them again, fortunately in much less time. However, I am not sure why the local build keep improving when I fixed all these links manually.

It might have do with my git commit. Will take a look tomorrow on a fresh brain.

File: CHANGELOG.md [hunk 3]
```diff
@@ -343,7 +343,7 @@ Querying with using `{__mimir_storage__="ephemeral"}` selector no longer works.
 ### Jsonnet
 
 * [CHANGE] Replaced the deprecated `policy/v1beta1` with `policy/v1` when configuring a PodDisruptionBudget. #3284
-* [CHANGE] [Common storage configuration](https://grafana.com/docs/mimir/v2.3.x/operators-guide/configure/configure-object-storage-backend/#common-configuration) is now used to configure object storage in all components. This is a breaking change in terms of Jsonnet manifests and also a CLI flag update for components that use object storage, so it will require a rollout of those components. The changes include: #3257
+* [CHANGE] [Common storage configuration](https://grafana.com/docs/mimir/v2.3.x/configure/configure-object-storage-backend/#common-configuration) is now used to configure object storage in all components. This is a breaking change in terms of Jsonnet manifests and also a CLI flag update for components that use object storage, so it will require a rollout of those components. The changes include: #3257
```
[the remark this answers is not in the document]
Comment (osg-grafana) [reply CHANGELOG.md]: @krajorama and I will handle this in a subsequent PR, likely today.

**Comment by osg-grafana** [comment]
@krajorama and I will handle changes to Helm chart documentation versions in a subsequent PR, likely today.

File: docs/sources/helm-charts/mimir-distributed/migration-guides/migrate-from-single-zone-with-helm.md (hunk 2 above)
Comment (osg-grafana) [reply docs/sources/helm-charts/mimir-distributed/migration-guides/migrate-from-single-zone-with-helm.md]: Will address version changes to Helm chart documentation in a subsequent PR.

**Comment by jdbaldry** [comment]
> > @jdbaldry, When I was doing this manually, I saw that link after link the build got better so I thought I was making progress. However, the doc validator seems to have me changing them again, fortunately in much less time. However, I am not sure why the local build keep improving when I fixed all these links manually.
> 
> It might have do with my git commit. Will take a look tomorrow on a fresh brain.

Ambiguous relref resolution can make the Hugo build improve but `doc-validator` is more strict on what it considers an acceptable relref.

For example, the relref `my-page` can be resolved by Hugo anywhere on the site, regardless of where you link from. However, the downside of this kind of relref is that it can fail in the full website build if there are two pages in the site that are both `my-page`. This will be increasingly common with consistent ToCs.

`doc-validator` will insist that you have relative relref resolution. Where Hugo will look anywhere in the site for `my-page`, `doc-validator` will only look in the current directory.

I am going to work on an improvement to `doc-validator` so that it properly supports strict relative resolution (relrefs that start with `./` or `../`) and partial URI resolution (relrefs that start with `/`) but rejects the potentially ambiguous resolution for any other relrefs.

**Review by jdbaldry — APPROVED** [review]
I see no errors when a website build with:

```console
$ $ ~/ext/grafana/technical-documentation/scripts/make-docs oncall incident mimir:v2.7.x grafana:v9.4:grafana:v9.4.x/docs/sources grafana-cloud "arbitrary:${HOME}/ext/grafana/website/scripts/docs/versions.js:/hugo/scripts/docs/versions.js" "arbitrary:${HOME}/ext/grafana/website/layouts/shortcodes/docs:/hugo/layouts/shortcodes/docs"
NOTE Serving '/home/jdb/ext/grafana/oncall/docs/sources' (branch 'main') at URL http://localhost:3002/docs/oncall/latest/
NOTE Serving '/home/jdb/ext/grafana/incident/docs/sources' (branch 'main') at URL http://localhost:3002/docs/incident/latest/
NOTE Serving '/home/jdb/ext/grafana/mimir/docs/sources/mimir' (branch 'osg-grafana/2023-03-13-move-configure-topics-to-top-level-of-toc') at URL http://localhost:3002/docs/mimir/v2.7.x/
NOTE Serving '/home/jdb/ext/grafana/grafana/v9.4.x/docs/sources' (branch 'jdb/2023-03-fix-website-build-errors-v9.4.x') at URL http://localhost:3002/docs/grafana/v9.4/
NOTE Serving '/home/jdb/ext/grafana/cloud-docs/docs/sources' (branch 'jdb/2023-03-fix-doc-validator') at URL http://localhost:3002/docs/grafana-cloud/
NOTE Mounting '/home/jdb/ext/grafana/website/scripts/docs/versions.js' at '/hugo/scripts/docs/versions.js'
NOTE Mounting '/home/jdb/ext/grafana/website/layouts/shortcodes/docs' at '/hugo/layouts/shortcodes/docs'

grafana -> v9.4
mimir -> v2.7.x
content/docs/grafana/v9.4/tutorials/build-a-data-source-backend-plugin -> content/tutorials/build-a-data-source-backend-plugin
content/docs/grafana/v9.4/tutorials/build-a-data-source-plugin -> content/tutorials/build-a-data-source-plugin
content/docs/grafana/v9.4/tutorials/build-an-app-plugin -> content/tutorials/build-an-app-plugin
content/docs/grafana/v9.4/tutorials/build-a-panel-plugin -> content/tutorials/build-a-panel-plugin
content/docs/grafana/v9.4/tutorials/build-a-panel-plugin-with-d3 -> content/tutorials/build-a-panel-plugin-with-d3
content/docs/grafana/v9.4/tutorials/build-a-streaming-data-source-plugin -> content/tutorials/build-a-streaming-data-source-plugin
content/docs/grafana/v9.4/tutorials/create-alerts-from-flux-queries -> content/tutorials/create-alerts-from-flux-queries
content/docs/grafana/v9.4/tutorials/create-users-and-teams -> content/tutorials/create-users-and-teams
content/docs/grafana/v9.4/tutorials/grafana-fundamentals -> content/tutorials/grafana-fundamentals
content/docs/grafana/v9.4/tutorials/iis -> content/tutorials/iis
content/docs/grafana/v9.4/tutorials/install-grafana-on-raspberry-pi -> content/tutorials/install-grafana-on-raspberry-pi
content/docs/grafana/v9.4/tutorials/integrate-hubot -> content/tutorials/integrate-hubot
content/docs/grafana/v9.4/tutorials/provision-dashboards-and-data-sources -> content/tutorials/provision-dashboards-and-data-sources
content/docs/grafana/v9.4/tutorials/run-grafana-behind-a-proxy -> content/tutorials/run-grafana-behind-a-proxy
content/docs/grafana/v9.4/tutorials/stream-metrics-from-telegraf-to-grafana -> content/tutorials/stream-metrics-from-telegraf-to-grafana
content/docs/grafana-cloud/tutorials/grafana-fundamentals-cloud -> content/tutorials/grafana-fundamentals-cloud
content/docs/grafana-cloud/tutorials/k8s-monitoring-app -> content/tutorials/k8s-monitoring-app
content/docs/mimir/v2.7.x/tutorials/play-with-grafana-mimir -> content/tutorials/play-with-grafana-mimir
content/docs/grafana/v9.4/alerting -> content/docs/grafana-cloud/alerting
content/docs/grafana/v9.4/developers/http_api -> content/docs/grafana-cloud/api-reference/http-api
content/docs/incident/latest -> content/docs/grafana-cloud/incident
hugo server -p 3003 --buildDrafts=false --buildFuture=false --bind 0.0.0.0 --noHTTPCache --minify --poll 700ms --renderToDisk -d dist --baseURL http://localhost:3002/ --appendPort=false
Start building sites … 
hugo v0.101.0-466fa43c16709b4483689930a4f9ac8add5c9f66+extended linux/amd64 BuildDate=2022-06-16T07:09:16Z VendorInfo=gohugoio

                   |  EN  |  DE  |  FR  |  ES  | PT-BR  
-------------------+------+------+------+------+--------
  Pages            | 2583 |   11 |   11 |   11 |    11  
  Paginator pages  |    0 |    0 |    0 |    0 |     0  
  Non-page files   |  141 |    0 |    0 |    0 |     0  
  Static files     | 1192 | 1192 | 1192 | 1192 |  1192  
  Processed images |    0 |    0 |    0 |    0 |     0  
  Aliases          | 3220 |    0 |    0 |    0 |     0  
  Sitemaps         |    2 |    1 |    1 |    1 |     1  
  Cleaned          |    0 |    0 |    0 |    0 |     0  

Built in 2407 ms
Watching for changes in /hugo/{content,data,i18n,layouts,package.json,static}
Use watcher with poll interval 700ms
Watching for config changes in /hugo/config/_default, /hugo/config/docs
Environment: "docs"
Serving pages from /hugo/dist
Running in Fast Render Mode. For full rebuilds on change: hugo server --disableFastRender
Web Server is available at http://localhost:3002/ (bind address 0.0.0.0)
Press Ctrl+C to stop
```

**Comment by osg-grafana** [comment]
> > > @jdbaldry, When I was doing this manually, I saw that link after link the build got better so I thought I was making progress. However, the doc validator seems to have me changing them again, fortunately in much less time. However, I am not sure why the local build keep improving when I fixed all these links manually.
> > 
> > 
> > It might have do with my git commit. Will take a look tomorrow on a fresh brain.
> 
> Ambiguous relref resolution can make the Hugo build improve but `doc-validator` is more strict on what it considers an acceptable relref.
> 
> For example, the relref `my-page` can be resolved by Hugo anywhere on the site, regardless of where you link from. However, the downside of this kind of relref is that it can fail in the full website build if there are two pages in the site that are both `my-page`. This will be increasingly common with consistent ToCs.
> 
> `doc-validator` will insist that you have relative relref resolution. Where Hugo will look anywhere in the site for `my-page`, `doc-validator` will only look in the current directory.
> 
> I am going to work on an improvement to `doc-validator` so that it properly supports strict relative resolution (relrefs that start with `./` or `../`) and partial URI resolution (relrefs that start with `/`) but rejects the potentially ambiguous resolution for any other relrefs.

Excellent. I am happy to review and test your solution, and thank you! :)

**Comment by osg-grafana** [comment]
I am about to commit changes to the aliases before squashing and merging.

**Comment by osg-grafana** [comment]
The aliases worked locally, and I tested them one-by-one.

**Pull request merged**